ext/AvizExt/viz/spatial.jl

```diff
@@ -17,20 +17,24 @@ Create a spatial choropleth figure.
 - `highlight` : Vector, stroke colors for each location
 - `centroids` : Vector{Tuple}, of lon and lats
 - `c_label` : String, label to use for color bar
+- `axis_opts` : Additional options to pass to adjust Axis attributes
+  See: https://docs.makie.org/v0.19/api/index.html#Axis
 """
-function create_map!(f, geodata, data, highlight, centroids, c_label)
+function create_map!(f, geodata, data, highlight, centroids, c_label, axis_opts)
     lon = first.(centroids)
     lat = last.(centroids)
 
+    axis_opts[:title] = get(axis_opts, :title, "Study Area")
```

Comment:
Does this replace "Study Area" with what is in axis_opts as the title?

Reply:
Julia documentation:
https://docs.julialang.org/en/v1/base/collections/#Base.get

Tooltip help in VS Code (displays on mouse hover):
![image](https://github.com/open-AIMS/ADRIA.jl/assets/4075136/c98d18ef-6ddf-448f-b8be-392496417344)

Help via the repl:

```julia-repl
julia> ?

help?> get
search: get get! getpid getkey getfield getindex getglobal getproperty gethostname get_color_palette get_zero_subnormals @get_attribute Segments SegmentationFault mergewith mergewith! RegexMatch linesegments LineSegments ArgumentError linesegments! MergeSort iter_segments @generated

  get(collection, key, default)

  Return the value stored for the given key, or the given default value if no mapping for the key is present.

  │ Julia 1.7
  │
  │  For tuples and numbers, this function requires at least Julia 1.7.

  Examples
  ≡≡≡≡≡≡≡≡≡≡

  julia> d = Dict("a"=>1, "b"=>2);

  julia> get(d, "a", 3)
  1

  julia> get(d, "c", 3)
  3
```

Reply:
Ah okey, cool thanks. Looks good to me so I'll merge.